src/components/application_manager/rpc_plugins/rc_rpc_plugin/include/rc_rpc_plugin/rc_rpc_plugin.h

```diff
@@ -65,7 +61,7 @@ class RCRPCPlugin : public plugins::RPCPlugin {
             app_mngr::rpc_service::RPCService& rpc_service,
             app_mngr::HMICapabilities& hmi_capabilities,
             policy::PolicyHandlerInterface& policy_handler,
-            resumption::LastState& last_state) OVERRIDE;
+            resumption::LastStateWrapperPtr last_state) OVERRIDE;
```

Comment:
re-add and deprecate old signature

Reply:
@jacobkeeler Done in 4fdae6f97e8445b4aebe5269c4163e028115f508